CHANGELOG.txt

```diff
@@ -18,7 +18,9 @@
       You should probably read the following releases notes for Django versions :
         - https://docs.djangoproject.com/en/3.2/releases/3.0/
         - https://docs.djangoproject.com/en/3.2/releases/3.1/
-    - The version of jQuery is now "2.2.4"
+    - The version of jQuery is now "3.6.0" (jQuery-migrate 3.3.2).
+        - The version of fullcalendar is now "3.10.2"
+        - The version of floatthead is now "2.2.1"
 
```

Comment:
Would be nice to add some hints about classical changes to do ; I have noticed:
 - .click() => .trigger('click')
 - .click(f) => .on('click', f)
 - change() =>.trigger('change')
 - $.trim(string) => $tring.trim()
 - .bind('X') => .on('X') (unbind('X') => off('X') )
 - $.isArray() => Array.isArray()
 - $.param() => creme.ajax.param()
 - .removeAttr('disabled') => .prop('disabled', false)

Reply:
Do you want a new `doc/en/Migration to jquery 3.rst` ?